website/src/controllers/api-v3/auth.js

```diff
@@ -124,7 +123,8 @@ api.registerLocal = {
 
     if (fbUser) {
       if (!fbUser.auth.facebook.id) throw new NotAuthorized(res.t('onlySocialAttachLocal'));
-      fbUser.auth.local = newUser;
+      fbUser.auth.local = newUser.auth.local;
+      fbUser.preferences = newUser.preferences;
```

Comment:
`preferences` are defined on line 119 of this same file. (let newUser = ... ). I cannot do fbUser.auth.local = newUser because newuser is mostly the auth.local part, not the whole user. So to conform to the schema (of user model), I put what is newUser here, into the fbUser.